pkg/ucp/frontend/controller/planes/listplanes.go

```diff
@@ -31,18 +31,18 @@ import (
 
 var _ armrpc_controller.Controller = (*ListPlanes)(nil)
 
-// ListPlanes is the controller implementation to get the list of UCP planes.
```

Comment:
This is a special case that we can't use the shared controller for. This is the `/planes` endpoint (list all planes regardless of type).